src/graph/create-type.ts

```diff
@@ -17,6 +17,7 @@ export const createType = ({ name, description, cover, properties }: Params) =>
   ops.push(...createDefaultProperties({ entityId: id, name, description, cover }));
 
   // set property "Types" to "Type"
+  assertValid(id);
```

Comment:
Might help to add a bit more to the JS Doc to denote the arguments and that this function can throw due to invalid id.

Reply:
good idea, will do!